vevestaX/vevesta.py

```diff
@@ -704,25 +702,27 @@ def dump(self, techniqueUsed, filename=None, message=None, version=None, showMes
             except Exception as e:
                 print('File not pushed to git')
 
-    def __EDA(self, fileName):
-        if isinstance(self.__data, pandasDataframe):
-            self.__EDAForPandas(fileName)
+    def EDA(self, df, fileName):
+        if isinstance(df, pandasDataframe):
+            self.__EDAForPandas(df, fileName)
 
-    def __EDAForPandas(self, fileName):
+    def __EDAForPandas(self, df, xlsxFilename):
```

Comment:
why has it been called XLSX filename since it will generate both pdf and excel files